_posts/en/newsletters/2025-04-18-newsletter.md

```diff
@@ -0,0 +1,111 @@
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+_New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates._
+
+- [Bitcoin Core 29.0rc3][] is a release candidate for the next major
+  version of the network's predominate full node.  Please see the
+  [version 29 testing guide][bcc29 testing guide].
+
+- [LND 0.19.0-beta.rc2][] is a release candidate for this popular LN
+  node.  One of the major improvements that could probably use testing
+  is the new RBF-based fee bumping for cooperative closes.
+
+<!-- FIXME:harding to update Thursday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [LDK #3593][] Implement a way to do BOLT 12 Proof of Payment
+
+- [BOLTs #1242][] Make payment_secret mandatory and ASSUMED
+
+## Correction
+
+Last week's newsletter [story][news349 ss] about SwiftSync contained
+several errors and confusing statements.
+
+- *No accumulator used:* we described SwiftSync as using a cryptographic
+  accumulator, which is incorrect.  An accumulator would allow testing
+  whether or not a transaction output (TXO) was a part of a set.
+  SwiftSync doesn't need to do that.  Instead, it adds a value
+  representing a TXO to an aggregate when the TXO is created and
+  subtracts the same value when the TXO is destroyed (spent).  After
+  performing this for all TXOs that were supposed to be spent before the
+  SwiftSync terminal block, the node checks that the aggregate is
+  zero---indicating that all created TXOs were later spent.
+
+- *Parallel block validation does not require assumevalid:* we described
+  one way parallel validation could work with SwiftSync, in which
+  scripts up to the terminal SwiftSync block were not
+  validated---similar to how Bitcoin Core works today during initial
+  sync with _assumevalid_.  However, previous scripts could be validated
+  with SwiftSync, although this would likely require changes to the
+  Bitcoin P2P protocol to optionally include extra data with blocks.
+  Bitcoin Core nodes already store this data for any block they also
+  store, so we don't think adding a P2P message extension would be
+  difficult if it was expected that a significant number of people
+  wanted to use SwiftSync with assumevalid disabled.
+
+- *Parallel block validation is for different reasons than Utreexo:*
+  we wrote that SwiftSync is able to parallel validate blocks for
```

Comment:
```suggestion
  we wrote that SwiftSync is able to parallel-validate blocks for
```
or "able to validate blocks in parallel for"
or "able to validate blocks concurrently for"